src/package.xml

```diff
@@ -29,7 +29,75 @@
         <name>CustomField</name>
     </types>
     <types>
-        <members>*</members>
+        <members>frField.address1</members>
+        <members>frField.address2</members>
+        <members>frField.age</members>
+        <members>frField.allocation</members>
+        <members>frField.allocations</members>
+        <members>frField.amount</members>
+        <members>frField.annualDonations</members>
+        <members>frField.anonymous</members>
+        <members>frField.campaignGoalId</members>
```

Comment:
Are we missing birth date in this list? assuming alphabetical it should be above this line

Reply:
I'm also fine if we change this back to a wildcard, because we know that we manage all of these records and we don't have to worry about adding new ones (like now where birth date is missing)